cx_Freeze/samples/msi_summary_data/hello.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import datetime
+import sys
+from sys import stdout
+
+stdout.write('Hello from cx_Freeze\n')
```

Comment:
Hi Marcelo--happy to change it.  The only reason I used stdout was because stdout was used in the "simple" sample, and I started from that code when making my sample.

Reply:
There are some too old samples from the epoch of py2 that need to be upgraded....

Reply:
I've fixed it for the msi_summary_data sample.  Happy to go through and fix some of the other samples at some point, if that would be welcome.

Reply:
If you can, thank you!